src/newsletterCommon/sharedComponents.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { useState } from 'react';
+import { css, ThemeProvider, keyframes } from '@emotion/react';
+import { Button, buttonThemeBrandAlt, SvgClock } from '@guardian/source-react-components';
+import type { TrackClick } from '../utils/tracking';
+
+import { news, neutral, body, textSans } from '@guardian/source-foundations';
+
+// Newsletter CTA button
+// -------------------------------------------------------
+const ctaStyles = {
```

Comment:
This is a very interesting question! I think in my ideal world I would like to keep the component the same for both banner and epic, but adapt the component so that it can receive styling data from each and respond accordingly. Like this 
<img width="414" alt="Screenshot 2023-03-01 at 10 52 44" src="https://user-images.githubusercontent.com/5357530/222119146-84f50f7a-f3c6-4a97-a806-26d75eed8250.png">

Reply:
Cool yeah that sounds like a good strategy. I think I tend to prefer a bit of duplication until that becomes an issue, but I don't feel super strongly so happy to go with your judgement!